Makefile

```
@@ -1,61 +1,176 @@
+# -*- coding: utf-8; mode: makefile-gmake -*-
 # Basic Makefile
 
 UUID = system-monitor@paradoxxx.zero.gmail.com
-BASE_MODULES = $(UUID)/extension.js $(UUID)/README* $(UUID)/metadata.json $(UUID)/prefs.js $(UUID)/stylesheet.css $(UUID)/convenience.js $(UUID)/compat.js $(UUID)/gpu_usage.sh
+INSTALLNAME = $(UUID)
+DESTDIR ?=
+
+BASE_MODULES = \
+  $(UUID)/extension.js \
+  $(UUID)/README* \
+  $(UUID)/metadata.json \
+  $(UUID)/prefs.js \
+  $(UUID)/stylesheet.css \
+  $(UUID)/convenience.js \
+  $(UUID)/compat.js \
+  $(UUID)/gpu_usage.sh
+
+# ---------
+# variables
+# ---------
+
 ifeq ($(strip $(DESTDIR)),)
-	INSTALLBASE = $(HOME)/.local/share/gnome-shell/extensions
+  INSTALLBASE = $(HOME)/.local/share/gnome-shell/extensions
+  SUDO=
 else
-	INSTALLBASE = $(DESTDIR)/usr/share/gnome-shell/extensions
+  INSTALLBASE = $(DESTDIR)usr/share/gnome-shell/extensions
```

This relies on the user realising/remembering to postfix `/` to their `DESTDIR=`.  There is no harm in having multiple `/`s so leaving this one in causes no harm and is defensive.
```suggestion
  INSTALLBASE = $(DESTDIR)/usr/share/gnome-shell/extensions
```

---

God point. I improved default handling of DESTDIR / see b431505 / is it OK?